src/schema/me/create_credit_card_mutation.js

```diff
@@ -43,9 +87,9 @@ export default mutationWithClientMutationId({
     },
   },
   outputFields: {
-    credit_card: {
-      type: CreditCardType,
-      resolve: credit_card => credit_card,
+    result: {
+      type: CreditCardMutationType,
+      resolve: result => result,
```

Comment:
Was any client (but mostly Eigen) using this part of the schema yet? Because this is a breaking change.